server/test/controllers/device/device.controller.test.js

```diff
@@ -109,6 +109,34 @@ describe('GET /api/v1/device_feature/aggregated_states', () => {
   });
 });
 
+describe('GET /api/v1/device_feature/:device_feature_selector/states', () => {
+  beforeEach(async function BeforeEach() {
+    this.timeout(1000);
+    await insertStates(1);
+  });
+  it('should get device feature states by selector', async () => {
+    const now = new Date();
+    const dateState = `${now.getUTCFullYear()}-${`0${now.getUTCMonth() + 1}`.slice(-2)}-${`0${now.getUTCDate()}`.slice(
+      -2,
```

Comment:
We use dayjs everywhere in Gladys, it might be simpler/more readable just to use a simple:


```
dayjs().format('YYYY-MM-DD')
```

No?